modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3113,17 +3099,19 @@ void ONNXImporter::parseCustomLayer(LayerParams& layerParams, const opencv_onnx:
 
 void ONNXImporter::parseQuantDequant(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
 {
-    CV_Assert(node_proto.input_size() == 3);
+    CV_Assert(node_proto.input_size() == 2 || node_proto.input_size() == 3);
     layerParams.type = (node_proto.op_type() == "QuantizeLinear") ? "Quantize" : "Dequantize";
+    Mat scale = getBlob(node_proto, 1);
+    int8_t zeropoint = node_proto.input_size() == 3 ? getBlob(node_proto, 2).at<int8_t>(0) : 0;
```

Comment:
Per spec, zeropoint should be of the same shape as scale, which can be either 0-d(per Tensor) or 1-d(per axis), but we don't support the latter: `CV_CheckEQ(scale.total(), 1, "")` should suffice.